ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -136,6 +138,25 @@ static ConfirmationFunction SimpleConfirmationFunction() {
   return cf;
 };
 
+// Check if op is supported by backend using is_supported API
+static Status IsSupportedByBackend(
+    const Node* node, const ng::runtime::Backend* op_backend,
+    std::map<std::string, std::set<shared_ptr<ng::Node>>>& TFtoNgraphOpMap,
+    bool& is_supported) {
+  is_supported = true;
+
+  auto ng_op = TFtoNgraphOpMap[node->type_string()];
```

Comment:
Check if item exists in the map, else return bad status

Reply:
using map.at() preferred instead of []

Reply:
> using map.at() preferred instead of []

why?

Reply:
[] adds a new key to the map if it does not exist
.at() checks if the key exists otherwise throws an error
http://www.cplusplus.com/reference/map/map/

Reply:
thanks, will make the change